src/monitor.cpp

```diff
@@ -161,6 +161,28 @@ unsigned long mTriggerUpdate = 0;
 unsigned long initialMillis = millis();
 unsigned long initialTime = 0;
 
+void getTime(unsigned long* currentHours, unsigned long* currentMinutes, unsigned long* currentSeconds){
+  
+  //Check if need an NTP call to check current time
+  if((mTriggerUpdate == 0) || (millis() - mTriggerUpdate > UPDATE_PERIOD_h * 60 * 60 * 1000)){ //60 sec. * 60 min * 1000ms
+    if(WiFi.status() == WL_CONNECTED) {
+        timeClient.update(); //NTP call to get current time
```

Comment:
There was recent MR with error handling on time update. Would be nice to have it here too
`if(timeClient.update()) mTriggerUpdate = millis();`

Reply:
Yes you are right, why was this not seen by github

Reply:
Because it is new code. But not a problem. It's easy to add it.

Reply:
Yes, I added now, I'll merge it